routes/admin_nft.go

```diff
@@ -295,8 +305,13 @@ func (fes *APIServer) AdminUpdateNFTDrop(ww http.ResponseWriter, req *http.Reque
 					"AdminUpdateNFTDrop: Cannot add non-NFT to drop: %v", postHash.String()))
 				return
 			}
-
-			updatedDropEntry.NFTHashes = append(updatedDropEntry.NFTHashes, postHash)
+			if BlockHashIn(updatedDropEntry.NFTHashes, postHash) {
+				_AddBadRequestError(ww, fmt.Sprintf(
+					"AdminUpdateNFTDrop: Cannot add duplicate NFT to drop: %v", postHash.String()))
+				return
+			} else {
+				updatedDropEntry.NFTHashes = append(updatedDropEntry.NFTHashes, postHash)
+			}
```

Comment:
```suggestion
			}
			updatedDropEntry.NFTHashes = append(updatedDropEntry.NFTHashes, postHash)
```

Since we return above, we don't need an else.